### PR TITLE
Initial PR (Bug Prediction)

### PR DESCRIPTION
**Pull request by fletna**
Added a description of paper selection and a bibliography.

### CODE REVIEW

File: bug_prediction.Rmd [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+# Bug Prediction
+
+## Identification of papers
+
+We started by studying the initial 6 seed papers [1] - [6] which helped us identify the key aspects of the bug prediction field we wanted to focus on. We decided to investigate the following questions: which metrics and which prediction models are predominantly used, how are the prediction models evaluated and are the current developments in the field able to provide actionable tools for developers.
+
+Our searches were based on the following elements:
+
+1. Keyword search using search engines (Scopus, ACM Digital Library, IEEE Explorer). The search query was constructed so that the paper title had to contain the phrase bug prediction, but also the other more general variants used in literature: *bug/defect/fault prediction*. The title also had to contain at least one of following keywords: *metrics*, *models*, *validation*, *evaluation*, *developers*. To remain within the bug prediction field we required *software* to appear in the abstract.
```
Comment (Inventitech) [inline bug_prediction.Rmd]: This line seems to be awfully long?

Comment (Inventitech) [inline bug_prediction.Rmd]: From my own experience, software might be a really hard requirement (for example, my PhD thesis title originally didn't contain a reference to that) and remove some useful papers

File: bug_prediction.Rmd [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: "Bug Prediction"
+author: "Guillem Bonet, Ana Semrov, Joost Wooning"
+date: "14 September 2018"
+output: html_document
```
Comment (Inventitech) [inline bug_prediction.Rmd]: We are not only publishing to HTML ...

File: book.bib [hunk 3]
```diff
@@ -1,12 +1,431 @@
...
+doi = {10.1109/TSE.2005.112},
+issn = {0098-5589},
+journal = {IEEE Transactions on Software Engineering},
+month = {oct},
+number = {10},
+pages = {897--910},
+title = {{Empirical validation of object-oriented metrics on open source software for fault prediction}},
+url = {http://ieeexplore.ieee.org/document/1542070/},
+volume = {31},
+year = {2005}
```
Comment (demasterr) [inline book.bib]: You might want to be careful with this reference as it is already 13 years old.

File: bug_prediction.Rmd [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+## Identification of papers
+
+We started by studying the initial 6 seed papers [1] - [6] which helped us identify the key aspects of the bug prediction field we wanted to focus on. We decided to investigate the following questions: which metrics and which prediction models are predominantly used, how are the prediction models evaluated and are the current developments in the field able to provide actionable tools for developers.
+
+Our searches were based on the following elements:
+
+1. Keyword search using search engines (Scopus, ACM Digital Library, IEEE Explorer). The search query was constructed so that the paper title had to contain the phrase bug prediction, but also the other more general variants used in literature: *bug/defect/fault prediction*. The title also had to contain at least one of following keywords: *metrics*, *models*, *validation*, *evaluation*, *developers*. To remain within the bug prediction field we required *software* to appear in the abstract.
+
+2. Filtering search results by publication date. We excluded papers older than 10 years; that is, published before 2008.
```
Comment (demasterr) [inline bug_prediction.Rmd]: Also see my other comment. You have one paper before 2008 in your BibTeX.

File: bug_prediction.Rmd [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Bug Prediction
+
+## Identification of papers
+
+We started by studying the initial 6 seed papers [1] - [6] which helped us identify the key aspects of the bug prediction field we wanted to focus on. We decided to investigate the following questions: which metrics and which prediction models are predominantly used, how are the prediction models evaluated and are the current developments in the field able to provide actionable tools for developers.
```
Comment (qurben) [inline bug_prediction.Rmd]: Maybe find a way to actually reference papers [1] - [6].

**Pull request closed without merging**